stacks/spica/src/bucket/services/bucket.service.ts

```diff
@@ -66,10 +67,19 @@ export class BucketService {
       .pipe(tap(_ => this.store.dispatch(new fromBucket.Update(id, changes))));
   }
 
-  patchIndexes(buckets: Bucket[]): Promise<Bucket[]> {
+  patchBucketWithoutStore(id: string, changes: object) {
+    return this.http.patch<Bucket>(`api:/bucket/${id}`, changes, {
+      headers: new HttpHeaders().set("Content-Type", "application/merge-patch+json")
+    });
+  }
```

Comment:
**patchBucket** and **patchBucketWithoutStore** duplicate the patch request. I would add **sendPatchRequest** method that sends only patch requests and use this method in the patchBucket and patchBucketMany methods.